packages/kbn-scout/src/common/services/clients.ts

```diff
@@ -29,7 +29,7 @@ function createClientUrlWithAuth({ serviceName, url, username, password, log }:
   return clientUrl.toString();
 }
 
-export function createEsClient(config: ScoutServerConfig, log: ToolingLog) {
+export function createEsClient(config: ScoutTestConfig, log: ToolingLog) {
```

Comment:
Renamed to differentiate servers configuration and Scout test configuration, that used by Playwright fixtures/tests.